exploration/number-selection.md

```diff
@@ -117,7 +117,16 @@ The function `:plural` operates identically to `:number`, except:
 
 The function `:ordinal` operates identically to `:number`, except:
 - the `select` type of this selector is always `ordinal`
-  (This function is valid as a formatter)
+- formatting behavior of this function is implementation defined
+
+> [!NOTE]
+> CLDR data has known shortcomings related to automatic formatting
```

Comment:
```suggestion
> CLDR data was designed to only format numeric ordinals, using a similar mechanism as plural selection, because the suffixes tend to need inflection.
```

Reply:
See my comment on a different thread; I've been thinking all the time about the selectOrdinal, which **has** sufficient coverage to support inclusion. (I'm not as sure about the *formatting* ordinal; I think it is supported in about half the coverage=modern locales.)

I'm ok with keeping ordinal out of the tech review, and I don't even think we need to cite a reason for that; we could just say that it is less used.

Reply:
This PR is about number _selection_. I don't see a reason to remove `:ordinal` from that. Formatting is something else.